src/App.js

```diff
@@ -14,28 +14,52 @@ class App extends React.Component {
       output: "",
       input: "",
       buttonText: "Submit",
-      description: "Description"
+      description: "Description",
+      show_example_form: false,
     };
 
     // Bind the event handlers
-    this.handleChange = this.handleChange.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
     this.handleClick = this.handleClick.bind(this);
   }
 
   componentDidMount() {
     // Call API for the UI params
     axios
       .get(UI_PARAMS_API_URL)
-      .then(({ data: { placeholder, button_text, description } }) => {
+      .then(({ data: { placeholder, button_text, description, show_example_form } }) => {
         this.setState({
           input: placeholder,
           buttonText: button_text,
-          description: description
+          description: description,
+          show_example_form: show_example_form,
+          examples: [],
         });
       });
   }
 
-  handleChange(e) {
+  handleExampleChange = (index, field) => e => {
+    let examples = [...this.state.examples];
+    examples[index][field] = e.target.value;
+    this.setState({ examples: examples });
+  }
+
+ handleExampleDelete = (index) => e => {
```

Comment:
helpful but somewhat misleading since deleting the example doesn't actually delete it from the gpt instance on the backend